scripts/run_lighthouse_tests.py

```diff
@@ -31,8 +31,20 @@
 
 WEBPACK_BIN_PATH = os.path.join(
     common.CURR_DIR, 'node_modules', 'webpack', 'bin', 'webpack.js')
+LIGHTHOUSE_MODE_PERFORMANCE = 0
```

Comment:
Please use 'performance', 'accessibility', 'dev', 'prod' as constants.

If these show up in an error message, we want them to be readable.